pyomo/contrib/doe/fim_doe_tutorial.ipynb

```diff
@@ -0,0 +1,2017 @@
+      "   4  0.0000000e+00 3.49e-10 2.83e-08  -2.5 1.09e-02    -  1.00e+00 1.00e+00h  1\n",
+      "\n",
+      "Number of Iterations....: 4\n",
+      "\n",
+      "                                   (scaled)                 (unscaled)\n",
+      "Objective...............:   0.0000000000000000e+00    0.0000000000000000e+00\n",
+      "Dual infeasibility......:   0.0000000000000000e+00    0.0000000000000000e+00\n",
+      "Constraint violation....:   3.4924596548080444e-10    3.4924596548080444e-10\n",
+      "Complementarity.........:   0.0000000000000000e+00    0.0000000000000000e+00\n",
+      "Overall NLP error.......:   3.4924596548080444e-10    3.4924596548080444e-10\n",
+      "\n",
+      "\n",
+      "Number of objective function evaluations             = 5\n",
+      "Number of objective gradient evaluations             = 5\n",
+      "Number of equality constraint evaluations            = 5\n",
+      "Number of inequality constraint evaluations          = 0\n",
+      "Number of equality constraint Jacobian evaluations   = 5\n",
+      "Number of inequality constraint Jacobian evaluations = 0\n",
+      "Number of Lagrangian Hessian evaluations             = 4\n",
+      "Total CPU secs in IPOPT (w/o function evaluations)   =      0.339\n",
+      "Total CPU secs in NLP function evaluations           =      0.014\n",
+      "\n",
+      "EXIT: Optimal Solution Found.\n",
+      "Existed information has been added.\n",
+      "Warning: FIM is near singular.\n",
+      "The condition number is: 31964777151094.688 ;\n",
+      "A condition number bigger than  1000000000000.0  is considered near singular.\n",
+      "FIM: [[  8.10994198   0.66261758 -25.28573045  -3.99722296]\n",
+      " [  0.66261758   6.51538037  -2.06594927 -39.30405278]\n",
+      " [-25.28573045  -2.06594927  78.83757565  12.4627842 ]\n",
+      " [ -3.99722296 -39.30405278  12.4627842  237.10182305]]\n",
+      "Trace: 330.5647210473526\n",
+      "Determinant: 3.182590499621547e-18\n",
+      "Condition number: 32006111569943.14\n",
+      "Minimal eigen value: 7.646435949699265e-12\n",
+      "Eigen values: [8.58320389e+01 2.44732682e+02 7.64643595e-12 1.98030178e-11]\n",
+      "Eigen vectors: [[ 0.30433403  0.02558873  0.87078696 -0.38394651]\n",
+      " [-0.01370187  0.16296194 -0.39918525 -0.902787  ]\n",
+      " [-0.94887345 -0.07978216  0.27928915 -0.12314403]\n",
+      " [ 0.08265733 -0.98306838 -0.06617237 -0.14965381]]\n",
+      "======problem solver output======\n",
+      "converged\n",
+      "WARNING: Implicitly replacing the Component attribute Obj (type=<class\n",
+      "    'pyomo.core.base.objective.ScalarObjective'>) on block unknown with a new\n",
+      "    Component (type=<class 'pyomo.core.base.objective.ScalarObjective'>). This\n",
+      "    is usually indicative of a modelling error. To avoid this warning, use\n",
+      "    block.del_component() and block.add_component().\n",
```

Comment:
I'm not sure what is generating this warning but this is a problem that needs to get fixed.

Reply:
This is because I use obj.deactivate(). This warning is fixed now.